components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -313,6 +327,9 @@ class InviteMembersStep extends React.PureComponent<Props, State> {
                                             <span>{this.state.emailError}</span>
                                         </>
                                     }
+                                    {(this.state.emailError && this.state.emails.length >= this.props?.subscriptionStats!.remaining_seats) &&
```

Comment:
Not sure I follow the usage of the non-null assertion here on `subscriptionStats`, I know we have moved the load to the root but shouldn't we do a null operator here to be safe instead and not assume it will not be null?

I would expect something like:  
```suggestion
                                    {(this.state.emailError && this.state.emails.length >= this.props.subscriptionStats?.remaining_seats) &&
```